python/commit_test.txt

```diff
@@ -0,0 +1,5 @@
+12/20/19 I made this file so that i can have something inconsequential to edit so that I can test pushing to github. - Preston
+
+Still shows that Minho made the commit on Github, despite changing the credentials in Windows Cred Manager and in the terminal.
+
+I changed the config file to reflect that the proper user was pushing to the git repository. It now correctly shows up on Git that I made the changes, and not Minho.
```

Comment:
This should be discarded in the PR since it was a Github learning thing rather than something important for the master branch

Reply:
yep